rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -797,10 +797,10 @@ namespace rsx
 
 		void reset_performance_overlay()
 		{
-			if (!g_cfg.misc.use_native_interface)
+			if (!g_fxo->is_init<rsx::overlays::display_manager>())
```

Comment:
this check seems redundant with your changes.
Can't you just use try_get in the check below?